erpnext/utilities/doctype/video/video.py

```diff
@@ -3,8 +3,51 @@
 # For license information, please see license.txt
 
 from __future__ import unicode_literals
-# import frappe
+import frappe
+import json
 from frappe.model.document import Document
+from six import string_types
+from pyyoutube import Api
 
 class Video(Document):
 	pass
+
+@frappe.whitelist()
+def get_video_stats(docname, youtube_id, update=True):
```

Comment:
**Suggestion:**

Move this inside the Video controller under `def get_stats(self): restofyourlogic` you can then use `frm.call` from the form instead.